embassy-stm32/src/rcc/g4.rs

```diff
@@ -15,6 +17,7 @@ pub const LSI_FREQ: Hertz = Hertz(32_000);
 pub enum ClockSrc {
     HSE(Hertz),
     HSI16,
+    PLL(PllSrc, PllM, PllN, PllClkDiv),
```

Comment:
Should this be `PLLR` instead of just `PLL`?

Additionally, is there an idea of how to change this to support configuring the PLL P and Q outputs for driving other peripherals?

Reply:
Hi, thanks for the review,

Yeah, I was unsure with the naming convention, since I partially copied the implementation from the l4 one as well as input from the ref-manual and the CubeIDE UI.

I was thinking, maybe we should strictly follow the CubeIDE/ref-manual naming as shown in the screenshot below:

![CleanShot 2023-06-07 at 19 57 13@2x](https://github.com/embassy-rs/embassy/assets/213276/829ad574-7299-42a1-8eb0-5c754d28b5f9)

This would mean changing `PLL` to `PLLCLK` (as refered on the System clock mux) and `PLLClkDiv` would be renamed to `PllR`.


As for the P and Q, I cheaped out a bit on the implementation since I didn't need them 😄 . But with the rename I think it would be quite easy to add in. (which I can also do in this PR)

Let me know what you think, I will do the required changes.

Reply:
Those changes sound good to me. My board can use the PLLQ output for CAN or USB related clocking so having those usable would be great.